core/domain/exp_services.py

```diff
@@ -1566,14 +1558,13 @@ def reject_suggestion(editor_id, thread_id, exploration_id):
         thread.put()
 
 
-def is_draft_version_valid(exp_id, user_id):
+def is_version_of_draft_valid(exp_id, version):
     """Checks if the draft version is the same as the latest version of the
-    exploration."""
+    exploration. Returns None if version is None."""
```

Comment:
This seems like a strange behaviour. Given the version is supplied by the calling function, it seems like that function should be responsible for knowing whether or not the version is None. I suggest getting rid of this feature, so that this function just returns a simple true / false.